packages/databyss-data/schemas/entrySchema.ts

```diff
@@ -0,0 +1,53 @@
+import { JSONSchema4 } from 'json-schema'
+
+export const entrySchema: JSONSchema4 = {
+  title: 'Entry',
+  type: 'object',
+  properties: {
+    _rev: {
+      type: 'string',
+    },
+    _revisions: {
+      type: 'object',
+      properties: {
+        start: {
+          type: 'number',
+        },
+        ids: {
+          type: 'array',
+          items: {
+            type: 'string',
+          },
+        },
+      },
+    },
+    _id: {
+      type: 'string',
+    },
+    type: {
```

Comment:
Following the mixin pattern, we should have a `blockSchema` mixin for schemas that share the `block` base class.
```
export const blockSchema = {
  type: {
    type: 'string',
  },
  text: {
    $ref: 'text',
  },
  account: {
    type: 'string',
  },
}
```